.goreleaser.yml

```diff
@@ -43,25 +39,28 @@ nfpms:
   -
     vendor: Apptio TechOps
     homepage: "https://github.com/apptio/kr8"
+    maintainer: 'Apptio, an IBM Company'
     description: "Opinionated configuration management tool for Kubernetes Cluster"
     license: MIT
     formats:
       - rpm
       - deb
     bindir: /usr/local/bin
 brews:
-  -
-    tap:
-      owner: apptio
-      name: homebrew-tap
-      token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
+  - name: kr8
+    url_template: "https://github.com/apptio/kr8/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+    url_headers:
+    - "Accept: application/octet-stream"
+    - 'Authorization: bearer #{ENV["HOMEBREW_TAP_GITHUB_TOKEN"]}'
+    download_strategy: CurlDownloadStrategy
```

Comment:
You'll need to add the `repository` field so that goreleaser knows where to push the recipe to.

Reply:
Added the repo field, I wasn't sure what to set the token key to so I used `PRIVATE_KEY_PATH`